package.json

```diff
@@ -20,8 +20,7 @@
   "scripts": {
     "start": "webpack-dev-server",
     "build": "webpack",
-    "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject",
+    "test": "jest",
```

Comment:
you can add this 
` "test:watch": "npm test -- --watch"`

Reply:
I saw it from one of the Jest tutorial, but decided to leave this out for now because I think it is up to the individual to enable this.